requirements.txt

```diff
@@ -6,6 +6,7 @@ django-cors-middleware==1.3.1
 django-test-without-migrations==0.4
 djangorestframework==3.5.3
 gunicorn==19.6.0
+pandas==0.19.2
```

Comment:
I'm using anaconda cause it was the only way I found through the problem. But I really don't think it's always necessary. But, if you really think it's better to close this PR, that's ok for me. :)